### PR TITLE
chore(agw): cleanup apt installs in agw Dockerfiles

### PR DESCRIPTION
**Pull request by wolfseb**
Signed-off-by: Sebastian Wolf <sebastian.wolf@tngtech.com>

## Summary

This PR sorts all installed packages alphanumerically for better maintainability. Duplicate packages are removed. An explicit cleanup is introduced after every install, where it was not done yet.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/docker/services/c/Dockerfile [hunk 1]
```diff
@@ -169,34 +169,34 @@ ENV TZ=Europe/Paris
...
   && rm -rf /var/lib/apt/lists/*
 
 RUN echo "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main" > /etc/apt/sources.list.d/magma.list
 RUN wget -qO - https://artifactory.magmacore.org:443/artifactory/api/gpg/key/public | apt-key add -
 RUN apt-get update && apt-get install -y \
   libopenvswitch \
-  openvswitch-datapath-dkms \
   openvswitch-common \
+  openvswitch-datapath-dkms \
   openvswitch-switch
```
Comment (jheidbrink) [inline lte/gateway/docker/services/c/Dockerfile]: Do we want a `&& rm -rf /var/lib/apt/lists/*` here?

File: lte/gateway/docker/services/python/Dockerfile [hunk 2]
```diff
@@ -117,12 +119,12 @@ RUN echo "deb https://packages.fluentbit.io/ubuntu/focal focal main" > /etc/apt/
...
 RUN apt-get update && apt-get install -y \
-  td-agent-bit \
+  bcc-tools \
   libopenvswitch \
   openvswitch-datapath-dkms \
   openvswitch-common \
   openvswitch-switch \
-  bcc-tools \
+  td-agent-bit \
   wireguard
```
Comment (jheidbrink) [inline lte/gateway/docker/services/python/Dockerfile]: Do we want a `&& rm -rf /var/lib/apt/lists/*` here?

**Comment by jheidbrink** [comment]
nitpick: personally I prefer meaningful `git blame` and better `git log -p` readability over alphabetically sorted lists. This is just to raise awareness of the advantage of not touching the lists, but it's fine with me if you prefer sorting anyway. :)

**Review by jheidbrink — APPROVED** [review]

File: lte/gateway/docker/services/c/Dockerfile [hunk 3]
```diff
@@ -49,63 +49,63 @@ RUN apt-get update && \
...
   oai-freediameter \
   oai-gnutls \
   oai-nettle \
-  magma-cpp-redis \
-  magma-libfluid \
-  grpc-dev \
   prometheus-cpp-dev \
-  liblfds710
-RUN rm /etc/apt/sources.list.d/magma.list
+  && rm /etc/apt/sources.list.d/magma.list
```
Comment (jheidbrink) [inline lte/gateway/docker/services/c/Dockerfile:109]: as discussed offlne: The RUN statement will remain untouched, and the rm will target `/var/lib/apt/lists/*` instead.

File: lte/gateway/docker/services/c/Dockerfile (hunk 1 above)
Comment (wolfseb) [reply lte/gateway/docker/services/c/Dockerfile]: I added it.

File: lte/gateway/docker/services/python/Dockerfile (hunk 2 above)
Comment (wolfseb) [reply lte/gateway/docker/services/python/Dockerfile]: I added it.

**Review by Neudrino — APPROVED** [review]

**Review by ShubhamTatvamasi — APPROVED** [review]

**Pull request merged**